recipe/bld.bat

```diff
@@ -1,5 +1,22 @@
-for /f "tokens=* usebackq" %%f in (`where nvcc`) do (set "dummy=%%f" && call set "CUDACXX=%%dummy:\=\\%%")
-echo "nvcc is %CUDACXX%, CUDA path is %CUDA_PATH%"
-echo "CMAKE_ARGS are %CMAKE_ARGS%"
-cmake -G Ninja -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% %CMAKE_ARGS% %SRC_DIR%
-ninja install
\ No newline at end of file
+:: Manually specify CUDA host compiler instead of relying on setup scripts
+:: https://gitter.im/conda-forge/conda-forge.github.io?at=5f8901b86c8d484be2898fc5
+
+set CUDACXX="%CUDA_HOME%\bin\nvcc.exe"
+set CUDAHOSTCXX="%CXX%"
+
+echo "C compiler is %CC%"
+%CC%
+echo "CXX compiler is %CXX%"
+%CXX%
+echo "CUDA path is %CUDA_PATH%"
+echo "CUDA home is %CUDA_HOME%"
+echo "CUDA compiler is %CUDACXX%"
+%CUDACXX% --version
+echo "CUDA host compiler is %CUDAHOSTCXX%"
+echo "CMAKE_ARGS is %CMAKE_ARGS%"
+cmake -G Ninja -DCMAKE_BUILD_TYPE=Release^
+  -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX%^
+  -DCMAKE_CUDA_COMPILER="%CUDACXX%"^
+  -DCMAKE_CUDA_HOST_COMPILER="%CUDAHOSTCXX%"^
```

Comment:
In CMake, if you pass it manually you need to replace the backslashes with forward slashes
```suggestion
  -DCMAKE_CUDA_COMPILER="%CUDACXX:\=/%"^
  -DCMAKE_CUDA_HOST_COMPILER="%CUDAHOSTCXX:\=/%"^
```